tests/py/casmutils/xtal/symmetry.py.in

```diff
@@ -7,51 +7,69 @@ import os
 
 input_file_dir = "@abs_top_srcdir@/tests/input_files"
 
+
 class SymmetryGroupsTest(unittest.TestCase):
 
     def setUp(self):
-        prim_fcc_ni_path=os.path.join(input_file_dir,"primitive_fcc_Ni.vasp")
-        self.prim_fcc_ni=cu.xtal.Structure.from_poscar(prim_fcc_ni_path)
-        self.cubic_lat=cu.xtal.Lattice([3,0,0],[0,3,0],[0,0,3])
-        self.tet_lat=cu.xtal.Lattice([3,0,0],[0,3,0],[0,0,4])
-        self.tol=1e-10;
-        hcp_mg_path=os.path.join(input_file_dir,"Mg_hcp.vasp")
-        self.hcp_mg=cu.xtal.Structure.from_poscar(hcp_mg_path)
-        almost_hcp_mg_path=os.path.join(input_file_dir,"distorted_Mg_hcp.vasp")
-        self.almost_hcp_mg=cu.xtal.Structure.from_poscar(almost_hcp_mg_path)
+        prim_fcc_ni_path = os.path.join(
+            input_file_dir, "primitive_fcc_Ni.vasp")
+        self.prim_fcc_ni = cu.xtal.Structure.from_poscar(prim_fcc_ni_path)
+        self.cubic_lat = cu.xtal.Lattice([3, 0, 0], [0, 3, 0], [0, 0, 3])
+        self.tet_lat = cu.xtal.Lattice([3, 0, 0], [0, 3, 0], [0, 0, 4])
+        self.tol = 1e-10
+        self.c = np.array([0, 0, 0])
```

Comment:
`c` and `d` aren't descriptive, and I'm not sure what they're supposed to represent. Are these variables even getting used?